contrib/pg_tde/t/001_basic.pl

```diff
@@ -34,7 +34,7 @@
 ok($cmdret == 0, "SELECT PGTDE VERSION");
 PGTDE::append_to_file($stdout);
 
-$rt_value = $node->psql('postgres', 'CREATE TABLE test_enc(id SERIAL,k INTEGER,PRIMARY KEY (id)) USING tde_heap_basic;', extra_params => ['-a']);
+$rt_value = $node->psql('postgres', 'CREATE TABLE test_enc(id SERIAL,k INTEGER,PRIMARY KEY (id)) USING tde_heap;', extra_params => ['-a']);
```

Comment:
Since this file is called basic shouldn't it be deleted or at least renamed?

Reply:
No, here it just means basic tests.